wrapping/itkElastixRegistrationMethod.wrap

```diff
@@ -1,4 +1,10 @@
 itk_wrap_class("itk::ElastixRegistrationMethod" POINTER)
-  itk_wrap_image_filter("${WRAP_ITK_REAL}" 2)
+  UNIQUE(dims "${ITK_WRAP_IMAGE_DIMS};4")
+  foreach(d ${dims})
+    foreach(t1 ${WRAP_ITK_SCALAR})
+      foreach(t2 ${WRAP_ITK_USIGN_INT} ${WRAP_ITK_SIGN_INT})
+        itk_wrap_template("${ITKM_I${t1}${d}}${ITKM_I${t2}${d}}"  "${ITKT_I${t1}${d}},${ITKT_I${t2}${d}}")
```

Comment:
`"${ITKM_I${t1}${d}}${ITKM_I${t2}${d}}"` -> `"I${ITKM_${t1}}${d}I${ITKM_${t2}}${d}"`

For `t1` is `F`, `d` is `4` it should be: `IF4IF4`.


`"${ITKT_I${t1}${d}},${ITKT_I${t2}${d}}"` -> `itk::Image< ${ITKT_${t1}}, ${d} >, itk::Image< ${ITKT_${t}}, ${d} >`

For `t1` is `F`, `d` is `4` it should be: `itk::Image< float, 4 >, itk::Image< float, 4 >`.